pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -84,6 +85,13 @@ func (p *IterationPrm) WithHandler(f func(addr *addressSDK.Address, data []byte)
 	return p
 }
 
+// WithLazyHandler sets a function to call on each object.
+// Second function will return file called if called. Otherwise, file is not read at all.
```

Comment:
does that line contain correct statements? if so, could you, please, clarify the meaning, i did not get it

Reply:
No, forgot to change

Reply:
Fixed